examples/portable/executor_runner/executor_runner.cpp

```diff
@@ -460,5 +560,52 @@ int main(int argc, char** argv) {
     ET_CHECK_MSG(status == Error::Ok, "Failed to save ETDump file.");
   }
 
+  if (bundle_io) {
+    // With bundled io we can check the result.
```

Comment:
This file gets copy pasted a ton as a basic example of a runner. Im pretty resistant to something like this which will tempt people to use it it as a tester. Im already not a huge fan of having the benchmarking stuff in here.